folder/file3.py

```diff
@@ -2,7 +2,7 @@
 
 def MyAnotherFunction():
     Variable = re.compile(r"123")
-    print(type(Variable))
+    print(type(Variable)    )
     return 0
```

Comment:
No spaces near round brackets are used in the source code, maybe it's better to remove spaces here.